src/CalculatorUITestFramework/StandardCalculatorPage.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+
+namespace CalculatorUITestFramework
+{
+    /// <summary>
+    /// This class contains the UI automation objects and helper methods available when the Calculator is in Standard Mode.
+    /// </summary>
+    public class StandardCalculatorPage
+    {
+        private WindowsDriver<WindowsElement> session => WinAppDriver.Instance.CalculatorSession;
+        public StandardOperatorsPanel StandardOperators = new StandardOperatorsPanel();
+        public MemoryPanel MemoryPanel = new MemoryPanel();
+        public HistoryPanel HistoryPanel = new HistoryPanel();
+        public NavigationMenu NavigationMenu = new NavigationMenu();
+        public WindowsElement Header
+        {
+            get
+            {
+                try
+                {
+                    return this.session.TryFindElementByAccessibilityId("Header");
+                }
+                catch
```

Comment:
Should have a comment here explaining why fallback to ContentPresenter is a reasonable thing to do. Also, is there a more specific exception type we can catch?

Reply:
I got this code from the original UI Tests project. I missed it when updating the other properties. It is updated to match the format of the other properties.